src/rendering/batcher/shared/const.ts

```diff
@@ -1 +1,3 @@
-export const MAX_TEXTURES = 16;
+import { maxRecommendedTextures } from '../../renderers/shared/texture/utils/maxRecommendedTextures';
+
+export const MAX_TEXTURES = maxRecommendedTextures();
```

Comment:
Do we even need this export? We are already caching maxTextureUnits, why not just call the maxRecommendedTextures inline?

Reply:
i think its a lot easier to read this way - its a good point though, i can remove the caching from the function?

Reply:
Yeah, I don't think the double cache is necessary.

Reply:
done!